yammbs/torsion/_store.py

```diff
@@ -302,37 +310,79 @@ def optimize_mm(
                     ),
                 )
 
-    def get_log_sse(
+    def get_rmsd(
+        self,
+        force_field: str,
+        molecule_ids: list[int] | None = None,
+        skip_check: bool = False,
+    ) -> RMSDCollection:
+        """Get the RMSD summed over the torsion profile."""
+        from openff.toolkit import Molecule
+
+        if not molecule_ids:
+            molecule_ids = self.get_molecule_ids()
+
+        if not skip_check:
+            self.optimize_mm(force_field=force_field)
+
+        rmsds = RMSDCollection()
+
+        for molecule_id in molecule_ids:
+            if molecule_id not in molecule_ids:
+                continue
+
+            qm_points = self.get_qm_points_by_molecule_id(id=molecule_id)
+            mm_points = self.get_mm_points_by_molecule_id(id=molecule_id, force_field=force_field)
+
+            molecule = Molecule.from_mapped_smiles(
+                self.get_smiles_by_molecule_id(molecule_id),
+                allow_undefined_stereo=True,
+            )
+
+            rmsds.append(
+                RMSD(
+                    id=molecule_id,
+                    rmsd=sum(get_rmsd(molecule, qm_points[key], mm_points[key]) for key in qm_points),
+                ),
+            )
+
+        return rmsds
+
+    def get_rmse(
         self,
         force_field: str,
         molecule_ids: list[int] | None = None,
         skip_check: bool = False,
-    ) -> LogSSECollection:
+    ) -> RMSECollection:
+        """Get the RMSD summed over the torsion profile."""
+
         if not molecule_ids:
             molecule_ids = self.get_molecule_ids()
 
         if not skip_check:
             self.optimize_mm(force_field=force_field)
 
-        log_sses = LogSSECollection()
+        rmses = RMSECollection()
 
-        for molecule_id in self.get_molecule_ids():
+        for molecule_id in molecule_ids:
             if molecule_id not in molecule_ids:
                 continue
```

Comment:
Ahh, this must be why this code was up above too. I think it's not necessary anymore, now that you assign `molecule_ids` earlier.